source/options.tsx

```diff
@@ -39,6 +42,12 @@ function reportStatus({error, text, scopes}: Status): void {
 		delete tokenStatus.dataset.validation;
 	}
 
+	// Toggle the ulists by token type (default to classic)
+	tokenType = tokenType ?? 'classic';
+	for (const ulist of select.all('[data-token-type]')) {
+		ulist.style.display = ulist.dataset.tokenType === tokenType ? '' : 'none';
+	}
```

Comment:
Used like this since I want to make `tokenType`'s default is 'classic' on status reporting, but below looks more better
```suggestion
	for (const ulist of select.all('[data-token-type]')) {
		ulist.style.display = (ulist.dataset.tokenType === tokenType ?? 'classic') ? '' : 'none';
	}
```